samples/unity-of-bugs/Assets/Scripts/MobileNativeSupport/NativeSupportButtons.cs

```diff
@@ -8,6 +9,18 @@
 
 public class NativeSupportButtons : MonoBehaviour
 {
+    [SerializeField] private GameObject _outOfMemoryButton;
+
+    private void Start()
+    {
+#if !UNITY_IOS
+        if (_outOfMemoryButton != null)
+        {
+            Destroy(_outOfMemoryButton);
+        }
+#endif
+    }
```

Comment:
Would this work?

```suggestion
#if UNITY_IOS
    [SerializeField] private GameObject _outOfMemoryButton;
#endif
```

Reply:
The GameObject `OOM Sample Button` exists and is hooked up for sending click events in the Editor. We just destroy it on any platform that is not iOS.